### PR TITLE
[MIRROR] Shifts vars off /mob to the correct subtype

### PR DESCRIPTION
**Pull request by ghost**
Original PR: https://github.com/tgstation/tgstation/pull/34773
--------------------
Moves some vars which had no function on /mob to the appropriate subtypes where they actually do things. 

Will clean more up in a later PR.

### CODE REVIEW

File: code/modules/mob/mob_defines.dm [hunk 1]
```diff
@@ -5,7 +5,8 @@
 	flags_1 = HEAR_1
 	hud_possible = list(ANTAG_HUD)
 	pressure_resistance = 8
-	use_tag = TRUE
+	use_tag = TRUE //HIPPIE NO IDE WHY?
```
Comment (Pyko1) [inline code/modules/mob/mob_defines.dm]: anyone know why?

File: code/modules/mob/mob_defines.dm [hunk 2]
```diff
@@ -22,39 +23,28 @@
 	*/
 	var/zone_selected = null
 
+	//hippie start
```
Comment (Pyko1) [inline code/modules/mob/mob_defines.dm]: also why?

Comment (ghost) [reply code/modules/mob/mob_defines.dm:26]: prolly remains of shitcode from the shitcoder we kicked

File: code/modules/mob/living/living_defines.dm [hunk 3]
```diff
@@ -5,6 +5,10 @@
 	hud_possible = list(HEALTH_HUD,STATUS_HUD,ANTAG_HUD)
 	pressure_resistance = 10
 
+	var/resize = 1 //Badminnery resize
```
Comment (Carbonhell) [inline code/modules/mob/living/living_defines.dm:8]: Ffs i wish this would be an /atom var...

Comment (GuyonBroadway) [reply code/modules/mob/living/living_defines.dm:8]: You should bitch at tg for that.

**Comment by JohnGinnane** [comment]
@Pyko1 @Chnkr is this also ready?

**Comment by Pyko1** [comment]
technically yes, even tho the hippie code part is most likely not needed.

**Comment by Pyko1** [comment]
would be best to remove the unneeded hippie code on a different pr tho, so id say its ready for merge.

**Review by JohnGinnane — APPROVED** [review]

**Pull request merged**